bottlerocket/testsys/src/run_sonobuoy.rs

```diff
@@ -46,13 +46,12 @@ pub(crate) struct RunSonobuoy {
     #[structopt(long, default_value = "e2e")]
     plugin: String,
 
-    /// The mode used for the sonobuoy test. One of `non-disruptive-conformance`,
-    /// `certified-conformance`, `quick`. Although the Sonobuoy binary defaults to
-    /// `non-disruptive-conformance`, we default to `quick` to make a quick test the most ergonomic.
+    /// The test mode passed to the sonobuoy E2E plugin. We default to `quick` to make a quick test
+    /// the most ergonomic.
```

Comment:
Nit: I feel like something is missing, probably?: 

```suggestion
    /// which is the most ergonomic.
```

Same in similar comments

Reply:
I think the original is correct. I means that we're making the quick test the most ergonomic (easiest) to run by making it the default.